src/routes/protocol.jsx

```diff
@@ -51,7 +51,7 @@ export default function Protocol() {
               </a>
               . Here you can interact with a{" "}
               <a
-                href={`${process.env.REACT_APP_MILKOMEDA_C1_EXPLORER}/address/${process.env.REACT_APP_DJED_ADDRESS}`}
+                href={`${process.env.REACT_APP_BC==='Milkomeda' ? process.env.REACT_APP_MILKOMEDA_C1_EXPLORER : process.env.REACT_APP_EXPLORER}/address/${process.env.REACT_APP_DJED_ADDRESS}`}
```

Comment:
This solution works, but it doesn't scale so well if, in the future, we have multiple blockchains and not only Milkomeda and Ethereum Classic.

A simpler and more scalable solution might be to rename both `REACT_APP_MILKOMEDA_C1_EXPLORER` and `REACT_APP_EXPLORER` to simply `REACT_APP_EXPLORER`.

Reply:
Now that you have renamed `REACT_APP_MILKOMEDA_C1_EXPLORER` to `REACT_APP_EXPLORER` in the env files, you need to fix this, otherwise the link will be undefined for milkomeda.

In general, remember: whenever you rename a variable, look for places in the code where you might be using that variable. Otherwise the renaming will break the proper functioning of the code.